PG.StarWarsGame.Files.MEG/PG.StarWarsGame.Files.MEG/Binary/File/Type/Definition/V1/MegFileNameTableRecord.cs

```diff
@@ -7,24 +10,26 @@
 using PG.Commons.Binary.File;
 using PG.Commons.Util;
 
-namespace PG.StarWarsGame.Files.MEG.Binary.File.Type.Definition
+namespace PG.StarWarsGame.Files.MEG.Binary.File.Type.Definition.V1
 {
     internal class MegFileNameTableRecord : IBinaryFile, ISizeable, IComparable
```

Comment:
would it make sense to use `struct` instead of `class` ?

Reply:
Whilst there technically may be (depending on Boxing, etc.) a few advantages to using a `struct` for a immutable `class` I'm not sure there is any gain from changing it, and I don't really have the time to actually test that properly, similar to the possible performance gain by using a `span` in the `StringUtility` class.